src/display/editor/toolbar.js

```diff
@@ -135,4 +135,79 @@ class EditorToolbar {
   }
 }
 
-export { EditorToolbar };
+class HighlightToolbar {
+  #buttons = null;
+
+  #toolbar = null;
+
+  #uiManager;
+
+  constructor(uiManager) {
+    this.#uiManager = uiManager;
+  }
+
+  #render() {
+    const editToolbar = (this.#toolbar = document.createElement("div"));
+    editToolbar.className = "editToolbar";
```

Comment:
Given another recent PR, should there perhaps be a suitable aria attribute added here?
```suggestion
    editToolbar.className = "editToolbar";
    editToolbar.setAttribute("role", "toolbar");
```